### DISCUSSION TITLE
Configuration Help

### DISCUSSION THREAD

**Discussion by Willmac16** [Willmac16/OctoPrint-GcodeLeveling#13] [General]
If you are having any trouble with plugin configuration, feel free to ask here.

**Comment by JkleSXB** [comment]
Hello,
Is there a way to use this plugin on an Artillery Sidewinder X1 without flashing the firmware ?
I tried it and it does not seem to work (echo:Unknown command: "G38.2 Z-2.5").
It would be great because I have a concave bed.
Thanks for your work anyway !
Regards,
Jean-Luc

**Reply by Willmac16** [reply]
I looked into it, and it sounds like the stock Sidewinder X1 doesn't have a probe that works with the glass bed/doesn't have one at all. In order to use the plugin you would need to manually measure the heights, so not using the `G38.2` command and autoprobing, but instead doing something like `G1 Z-0.1 F100` and checking with a piece of paper then entering the points in manually. If the machine won't let you go to negative Z heights then it sounds like you need to adjust the z-endstop switch and the bed so all points are at or above 0. [this thread may help with the bed adjustments](https://www.thingiverse.com/groups/artillery-sidewinder-x1-evnovo/forums/artillery-swx1/topic:41616)

**Reply by JkleSXB** [reply]
Thanks a lot for your reply and your time, I will try all this !
